### PR TITLE
blog: August newsletter

### PR DESCRIPTION
**Pull request by GregSutcliffe**
As usual. Did i miss anything?

### CODE REVIEW

File: _posts/2017-08-31-foreman-community-newsletter---august-2017.md [hunk 1]
```diff
@@ -0,0 +1,124 @@
...
+
+### Foreman 1.15.3 released
+
+1.15.3 was released on Aug 10th, containing fixes for over 25 issues. That's a
+ton of goodness, so you'll want to get upgrading! As always, the details are
+available on the [announce list][1_15_3] and as always you can report any
+future bugs on the [issue tracker][issues]. Thanks as always to the community
+for finding, reporting, and fixing these issues!
+
+### Katello 3.4.4 released
```
Comment (mmoll) [inline _posts/2017-08-31-foreman-community-newsletter---august-2017.md]: 3.4.5 also

File: _posts/2017-08-31-foreman-community-newsletter---august-2017.md [hunk 2]
```diff
@@ -0,0 +1,124 @@
...
+
+
```
Comment (mmoll) [inline _posts/2017-08-31-foreman-community-newsletter---august-2017.md]: superfluous line

File: _posts/2017-08-31-foreman-community-newsletter---august-2017.md [hunk 3]
```diff
@@ -0,0 +1,124 @@
...
+- foreman
+- newsletter
+- community
+excerpt: |
+  TODO
+---
+
+Another month, another newsletter. Time is a crazy thing :)
+
+--- new divide ---
```
Comment (mmoll) [inline _posts/2017-08-31-foreman-community-newsletter---august-2017.md]: ?

File: _posts/2017-08-31-foreman-community-newsletter---august-2017.md [hunk 4]
```diff
@@ -0,0 +1,124 @@
...
+
+After we went over through most pages of Foreman and its plugins, we would like
+to concentrate on more complex changes that requires a design or a discussion
+involving all interested parties. We'd like to incrementally discuss progress
+from both current design and implementation perspective as well as generic
+usability questions that pop up. This time we'll focus on various provisioning
+flows, look at first draft and discuss further steps. Demos will be recorded
+and streamed live so anyone from community can participate.
+
+* [Community Demo - 14th September]  (URL to be announced)
```
Comment (mmoll) [inline _posts/2017-08-31-foreman-community-newsletter---august-2017.md]: mhm

File: _posts/2017-08-31-foreman-community-newsletter---august-2017.md [hunk 5]
```diff
@@ -0,0 +1,124 @@
...
+In addition, the exisitng
+[foreman_monitoring](https://github.com/theforeman/foreman_monitoring) plugin
+has now been added to the DEB repos as well as the RPM repos - Debian users,
+rejoice!
+
+#### Updated plugins:
+
+- [smart_proxy_dns_infoblox](https://github.com/theforeman/smart_proxy_dns_infoblox) updated to 0.0.6
+- [smart_proxy_omaha](http://github.com/theforeman/smart_proxy_omaha) updated to 0.0.2
+- [smart_proxy_openscap](http://github.com/openscap/smart_proxy_openscap) updated to 0.6.5
```
Comment (mmoll) [inline _posts/2017-08-31-foreman-community-newsletter---august-2017.md]: these two should link to the https-URLs, IMHO

File: _posts/2017-08-31-foreman-community-newsletter---august-2017.md [hunk 6]
```diff
@@ -0,0 +1,124 @@
...
+layout: post
+title: Foreman Community Newsletter - August 2017
+date: 2017-08-31 12:29:15
+author: Greg Sutcliffe
+tags:
+- foreman
+- newsletter
+- community
+excerpt: |
+  TODO
```
Comment (mmoll) [inline _posts/2017-08-31-foreman-community-newsletter---august-2017.md]: still? ;)

File: _posts/2017-08-31-foreman-community-newsletter---august-2017.md (hunk 1 above)
Comment (GregSutcliffe) [reply _posts/2017-08-31-foreman-community-newsletter---august-2017.md]: Added

File: _posts/2017-08-31-foreman-community-newsletter---august-2017.md (hunk 3 above)
Comment (GregSutcliffe) [reply _posts/2017-08-31-foreman-community-newsletter---august-2017.md]: leftover rubbish, sorry

File: _posts/2017-08-31-foreman-community-newsletter---august-2017.md (hunk 4 above)
Comment (GregSutcliffe) [reply _posts/2017-08-31-foreman-community-newsletter---august-2017.md]: Formatting updated

File: _posts/2017-08-31-foreman-community-newsletter---august-2017.md [hunk 7]
```diff
@@ -0,0 +1,124 @@
...
+the mailing list!
+
+_(All Foreman events are now available via the [Foreman Calendar](/events))_
+
+### New / Updated plugins
+
+#### New plugins:
+
+ATIX.de have released a new plugin aimed at managing snapshots of machines on
+VMware vSphere. THis was actually released in July, but the RPM packaging is
```
Comment (orrabin) [inline _posts/2017-08-31-foreman-community-newsletter---august-2017.md]: THis

File: _posts/2017-08-31-foreman-community-newsletter---august-2017.md (hunk 5 above)
Comment (GregSutcliffe) [reply _posts/2017-08-31-foreman-community-newsletter---august-2017.md]: good catch, ty

File: _posts/2017-08-31-foreman-community-newsletter---august-2017.md (hunk 6 above)
Comment (GregSutcliffe) [reply _posts/2017-08-31-foreman-community-newsletter---august-2017.md]: Always :)

**Comment by GregSutcliffe** [comment]
Should add https://github.com/Katello/katello-packaging/tree/master/rubygem-foreman_scc_manager too?

EDIT: no, its 1.16 only. note for next time. Hat tip to @Klaas- for that

File: _posts/2017-08-31-foreman-community-newsletter---august-2017.md (hunk 7 above)
Comment (GregSutcliffe) [reply _posts/2017-08-31-foreman-community-newsletter---august-2017.md]: Ta, fixed

**Review by mmoll — APPROVED** [review]

**Comment by mmoll** [comment]
merged, thanks @GregSutcliffe!

**Pull request merged**